packages/pnpm/src/cmd/help.ts

```diff
@@ -33,6 +33,14 @@ const OPTIONS = {
     description: 'Skip staleness checks for cached data, but request missing data from the server',
     name: '--prefer-offline',
   },
+  store: {
+    description: 'The location where all the packages are saved on the disk',
+    name: '--store <path>',
+  },
+  virtualStoreDir: {
+    description: 'The directory that contains all the dependencies that are linked from the store (default is node_modules/.pnpm)',
```

Comment:
What isn't clear is what "all dependencies" means. Is it all dependencies listed in package.json, or all dependencies in the dependency tree? So depending on the answer to that question, I would suggest:

'The directory that contains links to the store for all of the dependencies **listed in package.json | in the dependency tree**.'

Reply:
It is what you call a dependency tree. But technically it is not a tree. It is a graph.

What about "The directory that contains links to the store for all of the dependencies (direct and indirect)

Reply:
More details added

Reply:
Technically it’s a graph, but most users think of it as a tree, especially since listing with full depth is represented as a tree.